airflow/providers/amazon/aws/hooks/appflow.py

```diff
@@ -0,0 +1,150 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import json
+import sys
+from datetime import datetime, timezone
+from time import sleep
+from typing import TYPE_CHECKING, List
+
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+
+if sys.version_info >= (3, 8):
+    from functools import cached_property
+else:
+    from cached_property import cached_property
```

Comment:
Now that all providers have their minimum Airflow version set to 2.2.0, you can use the functools compat instead! See #24582.
```suggestion
if sys.version_info >= (3, 8):
    from functools import cached_property
else:
    from cached_property import cached_property
```

Reply:
Done!